docker-compose-example.yaml

```diff
@@ -11,18 +11,18 @@ volumes:
 
 services:
   custom-exporter:
-    image: ghcr.io/arpa-network/custom-exporter:latest
+    image: custom-exporter:latest
     ports:
       - 8000:8000
     volumes:
-      - <your exporter config file path>:/app/exporter-config.yml
+      - ./custom-exporter/exporter-config.yml:/app/exporter-config.yml
     networks:
       - monitoring
   cloudwatch-exporter:
     image: prom/cloudwatch-exporter
     environment:
-      - AWS_ACCESS_KEY_ID=<YOUR_AWS_ACCESS_KEY>
-      - AWS_SECRET_ACCESS_KEY=<YOUR_AWS_SECRET_ACCESS_KEY>
+      - AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY_ID>
```

Comment:
I am not sure how you were planning on exposing this to the user, but you can consider filling in the AWS secret here directly.

Reply:
will add this in "clean up" phase once we limit the access to a smaller scope